connection.go

```diff
@@ -31,7 +33,7 @@ func NewConnection(session *Session, localClientID, remoteClientID string) (*Con
 		session:               session,
 		localClientID:         localClientID,
 		remoteClientID:        remoteClientID,
-		windowSize:            uint32(session.config.InitialConnectionWindowSize),
+		windowSize:            float64(session.sendWindowPacketCount) / float64(len(session.localClientIDs)),
```

Comment:
Add an `initialWindowSize` argument to NewConnection() because `len(session.localClientIDs)` is not always the number of connections.

Reply:
Yes, added